framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -554,6 +554,9 @@ public static String getSqlResult(ResultSet resultSet) throws SQLException {
               values.add(null);
             }
             break;
+          case Types.NULL:
+            values.add(null);
+            break;
```

Comment:
Hey @agozhiy , can you please have any framework code changes in a separate PR? I think it will help with maintenance.